Integrations/integration-dnstwist.yml

```diff
@@ -0,0 +1,76 @@
+description: Check dnstwist resolved domains variations to detect typo squatting,
+  phishing & corporate espionage
+configuration: []
+script:
+  script: |+
+    import subprocess, json
+
+    if demisto.command() == 'domain':
+
+        DOMAIN = demisto.args()['domain']
+        KEY = "domain-name"
+        KEYS_TO_COMPARE = ["whois-updated", "whois-created", "dns-a", "dns-mx"]
+
+        def get_dnstwist_result(domain):
+            res = subprocess.check_output(['/dnstwist/dnstwist.py', '-j', '-w', domain])
+            return json.loads(res)
+
+        def get_domain_to_info_map(dns_twist_result):
+            results = []
+            for x in dns_twist_result:
+                temp = {}
+                for k,v in x.items():
+                    if k in KEYS_TO_COMPARE:
+                        temp[k] = v
+                if temp:
+                    temp["domain-name"] = x["domain-name"]
+                    results.append(temp)
+            return results
+
+        dnstwist_result = get_dnstwist_result(DOMAIN) #contents
+        new_result = get_domain_to_info_map(dnstwist_result)
+        md = tableToMarkdown('dnstwist for domain - '  + DOMAIN, new_result)
+
+        DomainContext = new_result[0]
```

Comment:
why do you take only the results in index 0?
what about the rest? I think the point of this whole integration is the similar dns results and not the exact same one (otherwise it's better to use whois or domain tools...). SO, I think we will to store in the context all the similar incidents, under Domain.dnstwist context key. ask @michalgold